app/src/main/java/com/duckduckgo/app/browser/httpauth/WebViewHttpAuthStore.kt

```diff
@@ -31,11 +35,21 @@ interface WebViewHttpAuthStore {
     fun getHttpAuthUsernamePassword(webView: WebView, host: String, realm: String): WebViewHttpAuthCredentials?
     @UiThread
     fun clearHttpAuthUsernamePassword(webView: WebView)
+    @WorkerThread
+    suspend fun cleanHttpAuthDatabase()
 }
 
 class RealWebViewHttpAuthStore(
-    private val webViewDatabase: WebViewDatabase
+    private val webViewDatabase: WebViewDatabase,
+    private val databaseCleaner: DatabaseCleaner
 ) : WebViewHttpAuthStore {
+
+    init {
+        GlobalScope.launch {
+            databaseCleaner.changeJournalModeToDelete()
```

Comment:
Rather than manually deleting the journal, like @aitorvs suggested it works better to manually change the journal mode to delete. I am however not massively convinced about doing it here so happy to hear suggestion. I was thinking in a lifecyle observer oncreate of the app.

Reply:
yeah, I think a better place is in an `onCreate` app observer

Reply:
Changed!